src/MCPClient/lib/clientScripts/storeAIP.py

```diff
@@ -57,6 +58,21 @@ def get_upload_dip_path(aip_path):
     return os.path.sep + os.path.join(*new_aip_path)
 
 
+def rmtree_upload_dip_transitory_loc(package_type, unit_path):
+    """If a DIP has been stored but still exists in the DIP Upload watched
+    directory then it needs to be deleted.
+    """
+    if package_type == "DIP":
```

Comment:
Try to align the happy path to the left, e.g.:

```python
if package_type != "DIP":
    return
...
```

Related: https://medium.com/@matryer/line-of-sight-in-code-186dd7cdea88.